Makefile

```
@@ -1,6 +1,6 @@
 # Simple Easy Tasks - Makefile
 
-.PHONY: help build test test-verbose test-coverage clean lint run dev docker-build docker-run
+.PHONY: help build build-server build-cli build-pocketbase build-linux install test test-verbose test-coverage clean lint run dev docker-build docker-run
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)